cartodb/cartodb.py

```diff
@@ -138,7 +143,7 @@ def get_response_data(self, resp, parse_json=True):
                 return resp.json()
             return resp.content
         elif resp.status_code == requests.codes.bad_request:
-            raise CartoDBException(resp.json()['error'])
+            raise CartoDBException(resp.json()['errors'])
```

Comment:
Interesting... In fact when you run the test you get ```error``` in (for example):

```
{'error': ['relation "my_table_with_some_data" does not exist']}
```

but the error related to ```max_concurrent_import_count``` (when everything should work as expected) is:

```
{'errors': {'imports': 'invalid user max_concurrent_import_count (must be positive integer)'}}
```

I think that this should be standarized in the platform, don't you? Maybe we can make a list with the content of both ```error|s``` and report an issue upstreams.